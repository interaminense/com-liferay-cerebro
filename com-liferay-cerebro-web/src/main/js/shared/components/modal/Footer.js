import Component from 'metal-jsx';

class Footer extends Component {
	render() {
		return <div class="modal-footer">{this.props.children}</div>;
	}
}

export default Footer;