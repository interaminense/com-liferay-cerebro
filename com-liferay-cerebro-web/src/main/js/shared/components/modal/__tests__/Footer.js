import Footer from '../Footer';

describe('Modal Footer', () => {
	let component;

	afterEach(() => {
		if (component) {
			component.dispose();
		}
	});

	it('should render', () => {
		component = new Footer();

		expect(component).toMatchSnapshot();
	});
});