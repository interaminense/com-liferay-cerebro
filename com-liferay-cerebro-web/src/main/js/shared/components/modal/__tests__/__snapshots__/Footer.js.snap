// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Footer should render 1`] = `<div class="modal-footer"></div>`;
